boards/arm/nrf9160_pca10090/Kconfig.defconfig

```diff
@@ -30,4 +30,10 @@ config SPI_3
 
 endif # SPI
 
+config FLASH_LOAD_OFFSET
```

Comment:
If I have understood correctly how Kconfig works, this definition overwrites the default definition in Kconfig.zephyr, right? Do not we need to have the same content here, too? I.e. the content below (copy-pasted)
```
hex "Kernel load offset"
default 0
depends on HAS_FLASH_LOAD_OFFSET
```

Reply:
And same for _SIZE, of course

Reply:
Apparently not, we don't need to repeat any of that; we can just override the default.
I checked in `menuconfig` and everything looks okay, the help message is there too.
Perhaps @ulfalizer can comment?

Reply:
Defining a symbol/choice multiple times merges the definitions. You get a symbol with all the properties from the different locations, and the dependencies get ORed.

Take this definition for example:

```
config FOO
	int "foo"
	default 1 if A
	default 2 if B
	depends on X

...

config FOO
	default 3 if C
	depends on Y

...

config FOO
	default 4 if D
	default 5
	depends on Z
```

It's logically equivalent to this:

```
config FOO
	int "foo"
	default 1 if A
	default 2 if B
	default 3 if C
	default 4 if D
	default 5
	depends on X || Y || Z
```

There are some other subtleties as well. For example, if you put a prompt on more than one of the definitions, then you'll be able to see the symbol in multiple places in the `menuconfig` (wherever the prompt is visible). Internally, it's still a single symbol though.

Reply:
@ioannisg I think we can leave this as-is then.

Reply:
ok